workspaces/services/guacamole-azure-dsvm/.env.sample

```diff
@@ -0,0 +1,2 @@
+WORKSPACE_ID=__CHANGE_ME__
+GUACAMOLE_PARENT_SERVICE_ID=__CHANGE_ME__
```

Comment:
is there a reason this cannot be just `SERVICE_ID` - that way it's generic for all workspace service resources?